### DISCUSSION TITLE
Welcome to advanced-java Discussions!

### DISCUSSION THREAD

**Discussion by yanglbme** [doocs/advanced-java#206] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JosefinaNicola** [comment]
Hi, how are you doing?

My name is Josefina, I´am a recruiter from a company, we are looking foward Java Developer SR with Advanced English.
If you are interested you can send me an email: josefina.nicola@globallogic.com or we can connect on Linkedin: https://www.linkedin.com/in/josefina-nicola-3891b0151/

Greetings!
